requirements.txt

```diff
@@ -16,13 +18,19 @@ pymongo
 python-memcached
 cryptography
 boto3
+boto

```

Comment:
we have `boto3` why `boto` is also needed?

Reply:
there's an import for that `from boto import Config` in `object_core_lib` which I have removed as the config was causing an error while mocking, do we need to keep it? and how do we configure it with corelib